tests/python/contrib/test_ethosn/test_convert_equivalents.py

```diff
@@ -74,7 +74,7 @@ def before():
             relay.const(output_sc, "float32"),
             relay.const(output_zp, "int32"),
         )
-        composite = tei.make_ethosn_composite(expr, "ethos-n.qnn_mul")
```

Comment:
Same as one of the previously asked question: are the direct TVM mul -> NPU MUL test cases not needed any more?

Reply:
This was just because the composite name was changed to show more intent, TVM mul -> NPU mul was never supported